tests/test_extractsegmentationextractor.py

```diff
@@ -199,7 +199,10 @@ def test_extractor_get_images_dict(self):
             )[:]
 
         images_dict = self.extractor.get_images_dict()
-        self.assertEqual(len(images_dict), 3)
+        self.assertEqual(len(images_dict), 5)
+
+        self.assertEqual(images_dict["correlation"], None)
+        self.assertEqual(images_dict["mean"], None)
```

Comment:
Long-term TODO: I wish we didn't require these for all segmentation extractors, since everyone has a different way of summarizing images that don't clearly fit into notions of mean/correlation